include/zephyr/bluetooth/classic/avrcp.h

```diff
@@ -378,6 +408,37 @@ int bt_avrcp_tg_register_cb(const struct bt_avrcp_tg_cb *cb);
  */
 int bt_avrcp_tg_send_unit_info_rsp(struct bt_avrcp_tg *tg, uint8_t tid,
 				   struct bt_avrcp_unit_info_rsp *rsp);
+
+/** @brief Send the subunit info response.
+ *
+ *  This function is called by the application to send the subunit info response.
+ *
+ *  @param tg The AVRCP TG instance.
+ *  @param tid The transaction label of the response, valid from 0 to 15.
+ *  @param rsp The response for SUBUNIT INFO command.
+ *
+ *  @return 0 in case of success or error code in case of error.
+ */
+int bt_avrcp_tg_send_subunit_info_rsp(struct bt_avrcp_tg *tg, uint8_t tid,
+				      struct bt_avrcp_subunit_info_rsp *rsp);
+
+/** @brief Send AVRCP Pass Through response.
+ *
+ *  This function is called by the application to send the Pass Through response.
+ *
+ *  @param tg The AVRCP TG instance.
+ *  @param tid The transaction label of the response, valid from 0 to 15.
+ *  @param result The response code, see @ref bt_avrcp_rsp_t.
+ *  @param opid The user pass through operation id, see @ref bt_avrcp_opid_t.
+ *  @param state The button state, see @ref bt_avrcp_button_state_t.
+ *  @param payload The payload of the pass through response. Should not be NULL if len is not zero.
+ *  @param len The length of the payload.
+ *
+ *  @return 0 in case of success or error code in case of error.
+ */
+int bt_avrcp_tg_send_passthrough_rsp(struct bt_avrcp_tg *tg, uint8_t tid, bt_avrcp_rsp_t result,
+				     bt_avrcp_opid_t opid, bt_avrcp_button_state_t state,
+				     const uint8_t *payload, uint8_t len);
```

Comment:
```suggestion
int bt_avrcp_tg_send_passthrough_rsp(struct bt_avrcp_tg *tg, uint8_t tid, bt_avrcp_rsp_t result, struct net_buf *buf);
```
Add provide an API `struct net_buf *bt_avrcp_create_pdu(struct net_buf_pool *pool, k_timeout_t timeout);` for allocating net buffer.

And for passthrough_rsp, the format of the  `buf->data` is `struct bt_avrcp_passthrough_rsp`.

Reply:
Have provided the api in https://github.com/zephyrproject-rtos/zephyr/pull/91939/commits/f4bb4f4a0991f792427bb04348ae19522972bf25 PR